### ISSUE TITLE
Custom recipies crash

### ISSUE THREAD

**Issue by medsal15** [cyanobacterium/BaseMetals#7]
That was super fast. The 2 kinds of custom recipies (concern oredictionnary and doesn't concern it) crashed. Happened when I pressed "R" on a result of a custom recipe.

---- Minecraft Crash Report ----

WARNING: coremods are present:
  CCLCorePlugin (CodeChickenLib-1.8-1.1.2.115-universal.jar)
  NowWithRendering (redstonepaste-mc1.8-1.7.jar)
  CodeChickenCorePlugin (CodeChickenCore-1.8-1.0.5.34-universal.jar)
  NowWithRendering (hopperducts-mc1.8-1.4.5.jar)
  NEICorePlugin (NotEnoughItems-1.8-1.0.5.82-universal.jar)
  Init (RepairUnlimited1.8.jar)
  keysoverhaul (KeyBindingsOverhaul-LATEST-1.8.jar)
  LoadingPlugin (FastLeaveDecay-1.0.jar)
  BowInfinityFix (BowInfinityFix-1.8-rv1.jar)
  SRCore (SimpleRefinement-mc1.8-1.0.jar)
  CCCDeobfPlugin (unknown)
  CoreMod (Aroma1997Core-1.8-1.0.3.4.jar)
  LoadingPlugin (RandomThings-MC1.8-3.5.2.jar)
Contact their authors BEFORE contacting forge

// Oops.

Time: 21.06.15 18:35
Description: Updating screen events

java.lang.NullPointerException: Updating screen events
    at cyano.basemetals.registry.recipe.ArbitraryCrusherRecipe.getOutput(ArbitraryCrusherRecipe.java:41)
    at cyano.basemetals.registry.CrusherRecipeRegistry.getRecipesForOutputItem(CrusherRecipeRegistry.java:177)
    at cyano.basemetals.nei.CrusherRecipeHandler.loadCraftingRecipes(CrusherRecipeHandler.java:46)
    at codechicken.nei.recipe.TemplateRecipeHandler.loadCraftingRecipes(TemplateRecipeHandler.java:327)
    at cyano.basemetals.nei.CrusherRecipeHandler.loadCraftingRecipes(CrusherRecipeHandler.java:79)
    at codechicken.nei.recipe.TemplateRecipeHandler.getRecipeHandler(TemplateRecipeHandler.java:465)
    at codechicken.nei.recipe.GuiCraftingRecipe.openRecipeGui(GuiCraftingRecipe.java:20)
    at codechicken.nei.recipe.RecipeItemInputHandler.lastKeyTyped(RecipeItemInputHandler.java:23)
    at codechicken.nei.guihook.GuiContainerManager.lastKeyTyped(GuiContainerManager.java:308)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73869_a(GuiContainer.java)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_73869_a(GuiContainerCreative.java:303)
    at net.minecraft.client.gui.inventory.GuiContainer.public_func_73869_a(GuiContainer.java)
    at codechicken.nei.guihook.GuiContainerManager.callKeyTyped(GuiContainerManager.java)
    at codechicken.nei.guihook.GuiContainerManager.keyTyped(GuiContainerManager.java:477)
    at codechicken.nei.guihook.GuiContainerManager.handleKeyboardInput(GuiContainerManager.java:467)
    at net.minecraft.client.gui.inventory.GuiContainer.func_146282_l(GuiContainer.java)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:527)
    at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1670)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1028)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:345)
    at net.minecraft.client.main.Main.main(SourceFile:120)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at cyano.basemetals.registry.recipe.ArbitraryCrusherRecipe.getOutput(ArbitraryCrusherRecipe.java:41)
    at cyano.basemetals.registry.CrusherRecipeRegistry.getRecipesForOutputItem(CrusherRecipeRegistry.java:177)
    at cyano.basemetals.nei.CrusherRecipeHandler.loadCraftingRecipes(CrusherRecipeHandler.java:46)
    at codechicken.nei.recipe.TemplateRecipeHandler.loadCraftingRecipes(TemplateRecipeHandler.java:327)
    at cyano.basemetals.nei.CrusherRecipeHandler.loadCraftingRecipes(CrusherRecipeHandler.java:79)
    at codechicken.nei.recipe.TemplateRecipeHandler.getRecipeHandler(TemplateRecipeHandler.java:465)
    at codechicken.nei.recipe.GuiCraftingRecipe.openRecipeGui(GuiCraftingRecipe.java:20)
    at codechicken.nei.recipe.RecipeItemInputHandler.lastKeyTyped(RecipeItemInputHandler.java:23)
    at codechicken.nei.guihook.GuiContainerManager.lastKeyTyped(GuiContainerManager.java:308)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73869_a(GuiContainer.java)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_73869_a(GuiContainerCreative.java:303)
    at net.minecraft.client.gui.inventory.GuiContainer.public_func_73869_a(GuiContainer.java)
    at codechicken.nei.guihook.GuiContainerManager.callKeyTyped(GuiContainerManager.java)
    at codechicken.nei.guihook.GuiContainerManager.keyTyped(GuiContainerManager.java:477)
    at codechicken.nei.guihook.GuiContainerManager.handleKeyboardInput(GuiContainerManager.java:467)
    at net.minecraft.client.gui.inventory.GuiContainer.func_146282_l(GuiContainer.java)
    at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:527)

-- Affected screen --
Details:
    Screen name: net.minecraft.client.gui.inventory.GuiContainerCreative

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityPlayerSP['medsal15'/70, l='MpServer', x=857.50, y=4.00, z=1203.50]]
    Chunk stats: MultiplayerChunkCache: 617, 617
    Level seed: 0
    Level generator: ID 01 - flat, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: 867.00,4.00,1213.00 - World: (867,4,1213), Chunk: (at 3,0,13 in 54,75; contains blocks 864,0,1200 to 879,255,1215), Region: (1,2; contains chunks 32,64 to 63,95, blocks 512,0,1024 to 1023,255,1535)
    Level time: 31392 game time, 31392 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 30 total; [EntityHorse['Horse'/64, l='MpServer', x=937.72, y=4.00, z=1186.06], EntityPatronEffect['unknown'/13440, l='MpServer', x=857.50, y=4.00, z=1203.50], EntityHorse['Horse'/65, l='MpServer', x=933.59, y=4.00, z=1188.16], EntityRabbit['Rabbit'/66, l='MpServer', x=928.78, y=4.00, z=1185.56], EntityHorse['Horse'/67, l='MpServer', x=935.69, y=4.00, z=1189.63], EntityChicken['Chicken'/14, l='MpServer', x=778.50, y=4.00, z=1195.50], EntityChicken['Chicken'/15, l='MpServer', x=778.41, y=4.00, z=1200.41], EntityVillager['Villager'/22, l='MpServer', x=782.28, y=5.00, z=1280.97], EntityChicken['Chicken'/27, l='MpServer', x=789.09, y=4.00, z=1198.69], EntityVillager['Villager'/28, l='MpServer', x=799.69, y=5.00, z=1263.88], EntityVillager['Villager'/29, l='MpServer', x=798.94, y=5.00, z=1262.63], EntityVillager['Villager'/30, l='MpServer', x=799.28, y=5.00, z=1273.97], EntityVillager['Villager'/31, l='MpServer', x=798.28, y=5.00, z=1264.19], EntityVillager['Villager'/33, l='MpServer', x=807.31, y=5.00, z=1263.69], EntityVillager['Villager'/34, l='MpServer', x=805.78, y=4.00, z=1274.72], EntityPlayerSP['medsal15'/70, l='MpServer', x=857.50, y=4.00, z=1203.50], EntityVillager['Villager'/35, l='MpServer', x=811.50, y=6.00, z=1277.34], EntityVillager['Villager'/36, l='MpServer', x=812.34, y=5.00, z=1280.91], EntityVillager['Villager'/39, l='MpServer', x=830.31, y=5.00, z=1276.25], EntityChicken['Chicken'/45, l='MpServer', x=892.63, y=4.00, z=1196.53], EntityItem['item.item.egg'/46, l='MpServer', x=892.47, y=4.00, z=1196.22], EntityChicken['Chicken'/47, l='MpServer', x=893.50, y=4.00, z=1198.50], EntityItem['item.item.egg'/48, l='MpServer', x=894.34, y=4.00, z=1197.63], EntityChicken['Chicken'/49, l='MpServer', x=890.50, y=4.00, z=1201.50], EntityChicken['Chicken'/50, l='MpServer', x=902.31, y=4.00, z=1190.66], EntitySheep['Sheep'/59, l='MpServer', x=915.97, y=4.00, z=1131.63], EntitySheep['Sheep'/60, l='MpServer', x=933.72, y=4.00, z=1141.34], EntitySheep['Sheep'/61, l='MpServer', x=930.72, y=4.00, z=1134.50], EntitySheep['Sheep'/62, l='MpServer', x=930.50, y=4.00, z=1139.16], EntityHorse['Horse'/63, l='MpServer', x=936.66, y=4.00, z=1187.84]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Integrated singleplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:351)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2502)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:367)
    at net.minecraft.client.main.Main.main(SourceFile:120)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
    Minecraft Version: 1.8
    Operating System: Windows 8.1 (amd64) version 6.3
    Java Version: 1.8.0_45, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 233603976 bytes (222 MB) / 598736896 bytes (571 MB) up to 1060372480 bytes (1011 MB)
    JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.10 FML v8.0.99.99 Minecraft Forge 11.14.3.1468 44 mods loaded, 44 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.05} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJAAAA   FML{8.0.99.99} [Forge Mod Loader](forge-1.8-11.14.3.1468.jar) 
    UCHIJAAAA   Forge{11.14.3.1468} [Minecraft Forge](forge-1.8-11.14.3.1468.jar) 
    UCHIJAAAA   Aroma1997Core{1.0.3.4} [Aroma1997Core](Aroma1997Core-1.8-1.0.3.4.jar) 
    UCHIJAAAA   bowinfinityfix{rv1} [Bow Infinity Fix](minecraft.jar) 
    UCHIJAAAA   CodeChickenCore{1.0.5.34} [CodeChicken Core](minecraft.jar) 
    UCHIJAAAA   NotEnoughItems{1.0.5.82} [Not Enough Items](NotEnoughItems-1.8-1.0.5.82-universal.jar) 
    UCHIJAAAA   REPAIR_UNLIMITED{0.0.0} [Repair Unlimited](minecraft.jar) 
    UCHIJAAAA   Aroma1997CoreHelper{1.0.3.4} [Aroma1997Core|Helper](Aroma1997Core-1.8-1.0.3.4.jar) 
    UCHIJAAAA   Aroma1997sDimension{1.0} [Aroma1997's Dimensional World](Aroma1997s-Dimensional-World-1.8-1.1.1.0.jar) 
    UCHIJAAAA   AutoSapling{1.0} [AutoSapling](AutoSap-MC1.8-1.2.jar) 
    UCHIJAAAA   basemetals{1.3.3} [Base Metals](BaseMetals-1.3.3.jar) 
    UCHIJAAAA   BetterChests{1.2.0.0} [BetterChests](BetterChests-1.8-1.2.0.0.jar) 
    UCHIJAAAA   wonderfulwands{1.7.0} [Wonderful Wands](CyanosWonderfulWands-1.7.0.jar) 
    UCHIJAAAA   decorationmegapack{1.17} [Decoration Mega Pack](decorationmegapack-1.17-1.8-1446.jar) 
    UCHIJAAAA   FastLeaveDecay{1.0} [Fast Leave Decay](FastLeaveDecay-1.0.jar) 
    UCHIJAAAA   genuine-easapoc{1.3.3.7_a1} [Solar Apocalypse](genuine-ea-1.3.3.7_a1.jar) 
    UCHIJAAAA   genuine-ea{1.3.3.7_a1} [Elite Armageddon](genuine-ea-1.3.3.7_a1.jar) 
    UCHIJAAAA   glass_shards{1.3.1} [Glass Shards](glass_shards_mc1.8-1.3.1.jar) 
    UCHIJAAAA   iChunUtil{5.4.0} [iChunUtil](iChunUtil-5.4.0.jar) 
    UCHIJAAAA   GravityGun{5.0.0} [GravityGun](GravityGun-5.0.0.jar) 
    UCHIJAAAA   GuiltTrip{5.0.0} [GuiltTrip](GuiltTrip-5.0.0.jar) 
    UCHIJAAAA   hopperducts{1.4.5} [Hopper Ducts](hopperducts-mc1.8-1.4.5.jar) 
    UCHIJAAAA   IronChest{6.0.87.745} [Iron Chest](ironchest-1.8-6.0.87.745-universal.jar) 
    UCHIJAAAA   keysoverhaul{1.0} [Key Bindings Overhaul](KeyBindingsOverhaul-LATEST-1.8.jar) 
    UCHIJAAAA   LomLib{4.1.0} [LomLib](LomLib-1.8-4.1.0-universal.jar) 
    UCHIJAAAA   lucky{5.2.0} [Lucky Block](LuckyBlock_1-8-1_v5-2-0.jar) 
    UCHIJAAAA   LunatriusCore{1.1.2.26} [LunatriusCore](LunatriusCore-1.8-1.1.2.26-universal.jar) 
    UCHIJAAAA   MemoryCleaner{1.0} [Memory Cleaner](MemoryCleaner-1.4-mc.1.8.jar) 
    UCHIJAAAA   MINE_SPAWNER{1.0} [MineSpawner](MineSpawner_v1.0.1[1.8][Forge].jar) 
    UCHIJAAAA   ministrapp{1.8.0-4.0.0A} [Minestrappolation](Minestrappolation-1.8-4.0.0.nightly_70.jar) 
    UCHIJAAAA   PeacefulSurface{1.8.x-v1} [PeacefulSurface](PeacefulSurface-1.8.x-v1.jar) 
    UCHIJAAAA   pcb{1.4} [Portable Craft Bench](portablecraftbench-1.8-1.4.jar) 
    UCHIJAAAA   poweradvantage{1.0.2} [Power Advantage](PowerAdvantage-1.0.2.jar) 
    UCHIJAAAA   RandomThings{3.5.2} [Random Things](RandomThings-MC1.8-3.5.2.jar) 
    UCHIJAAAA   redstonepaste{1.7} [Redstone Paste](redstonepaste-mc1.8-1.7.jar) 
    UCHIJAAAA   AS_Ruins{14.8} [Ruins Spawning System](Ruins-1.8.jar) 
    UCHIJAAAA   screenshots{1.0.0} [Screenshots Enhanced](screenshots-1.0.0-1.8-hotfix.jar) 
    UCHIJAAAA   simplerefinement{1.0} [Simple Refinement](SimpleRefinement-mc1.8-1.0.jar) 
    UCHIJAAAA   Stackie{1.5.1.32} [Stackie](Stackie-1.8-1.5.1.32-universal.jar) 
    UCHIJAAAA   steamadvantage{1.1.1} [Steam Advantage](SteamAdvantage-1.1.1.jar) 
    UCHIJAAAA   chunkbase_villageinfo_ssp{2.1} [Village Info (SSP)](VillageInfoSSP-1.8-2.1.jar) 
    UCHIJAAAA   villagertweaks{2.2} [Villager Tweaks](VillagerTweaks[1.8]-2.2.jar) 
    UCHIJAAAA   Waila{1.6.0_B1} [Waila](Waila-1.6.0_B1_1.8.1.jar) 
    Loaded coremods (and transformers): 
CCLCorePlugin (CodeChickenLib-1.8-1.1.2.115-universal.jar)
  codechicken.lib.asm.ClassHeirachyManager
  codechicken.lib.asm.RenderHookTransformer
NowWithRendering (redstonepaste-mc1.8-1.7.jar)
  net.fybertech.nwr.NowWithRendering
CodeChickenCorePlugin (CodeChickenCore-1.8-1.0.5.34-universal.jar)
  codechicken.core.asm.InterfaceDependancyTransformer
  codechicken.core.asm.TweakTransformer
  codechicken.core.asm.DelegatedTransformer
  codechicken.core.asm.DefaultImplementationTransformer
NowWithRendering (hopperducts-mc1.8-1.4.5.jar)
  net.fybertech.nwr.NowWithRendering
NEICorePlugin (NotEnoughItems-1.8-1.0.5.82-universal.jar)
  codechicken.nei.asm.NEITransformer
Init (RepairUnlimited1.8.jar)
  repairunlimit.Transformer
keysoverhaul (KeyBindingsOverhaul-LATEST-1.8.jar)
  code.elix_x.coremods.keysoverhaul.core.KeysOverhaulTransformer
LoadingPlugin (FastLeaveDecay-1.0.jar)
  lumien.fastleavedecay.asm.ClassTransformer
BowInfinityFix (BowInfinityFix-1.8-rv1.jar)
  com.github.parker8283.bif.Transformer
SRCore (SimpleRefinement-mc1.8-1.0.jar)
  net.fybertech.simplerefinement.SRCore
CCCDeobfPlugin (unknown)

CoreMod (Aroma1997Core-1.8-1.0.3.4.jar)

LoadingPlugin (RandomThings-MC1.8-3.5.2.jar)
  lumien.randomthings.asm.ClassTransformer
    GL info: ' Vendor: 'Intel' Version: '4.0.0 - Build 10.18.10.3408' Renderer: 'Intel(R) HD Graphics'
    Launched Version: 1.8-Forge11.14.3.1468
    LWJGL: 2.9.1
    OpenGL: Intel(R) HD Graphics GL version 4.0.0 - Build 10.18.10.3408, Intel
    GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

```
Using VBOs: No
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
```

**Comment by medsal15** [comment]
Only happens with NEI. Using the recipies (almost) doesn't create any problem.

**Comment by DrPlantabyte** [comment]
Well that's a pretty inconvenient bug. I'll try to fix it soon.

**Comment by DrPlantabyte** [comment]
Could you post your config file? It is acting like you gave it a recipe with a null/invalid output (which is still a bug on my end, since that doesn't need to cause a crash).

**Comment by medsal15** [comment]
# Configuration file

##########################################################################################################

# hammer recipes

#--------------------------------------------------------------------------------------------------------#

# This section allows you to add your own recipes for the Crack Hammer (and other rock

# crushers). Recipes are specified in semicolon (;) delimited lists of formulas in the

# format modid:name#y->x*modid:name#y, where x is the number of items in a stack and y

# is the metadata value. Note that both x and y are optional, so you can use the

# formula modid:name->modid:name for most items/blocks.

# 

# All properties in this section will be parsed for formulas, regardless their name.

# This lets you organize your recipe lists for easier reading.

##########################################################################################################

"hammer recipes" {
    S:custom=ministrapp:biome_stone#0->ministrapp:biome_cobble#0;ministrapp:biome_stone#1->ministrapp:biome_cobble#1;ministrapp:biome_stone#2->ministrapp:biome_cobble#2;ministrapp:biome_stone#3->ministrapp:biome_cobble#3;ministrapp:biome_stone#4->ministrapp:biome_cobble#4;ministrapp:biome_stone#5->ministrapp:biome_cobble#5;ministrapp:biome_stone#6->ministrapp:biome_cobble#6;ministrapp:biome_stone#7->ministrapp:biome_cobble#7;ministrapp:biome_stone#8->ministrapp:biome_cobble#8;ministrapp:copper_ore->2_ministrapp:chunks#5;ministrapp:biome_copper->2_ministrapp:chunks#5;ministrapp:tin_ore->2_ministrapp:chunks#6;ministrapp:biome_tin->2_ministrapp:chunks#6;ministrapp:meurodite_ore->2_ministrapp:meurodite_gem;ministrapp:biome_meurodite->2_ministrapp:meurodite_gem;ministrapp:torite_ore->2_ministrapp:chunks#9;ministrapp:biome_torite->2_ministrapp:chunks#9;ministrapp:biome_redstone->4_minecraft:redstone;ministrapp:sunstone_ore->2_ministrapp:sunstone_shard;ministrapp:biome_sunstone->2_ministrapp:sunstone_shard;ministrapp:plutonium_ore->2_ministrapp:plutonium;ministrapp:biome_plutonium->2_ministrapp:plutonium;ministrapp:uranium_ore->2_ministrapp:uranium;ministrapp:biome_uranium->2_ministrapp:uranium;ministrapp:radiant_ore->2_ministrapp:radiant_quartz;ministrapp:biome_radiant->2_ministrapp:radiant_quartz;ministrapp:titanium_ore->2_ministrapp:chunks#10;ministrapp:biome_titanium->2_ministrapp:chunks#10;ministrapp:blazium_ore->2_ministrapp:blaze_shard;ministrapp:soul_ore->2_ministrapp:soul_gem;ministrapp:copper_block->9_ministrapp:chunks#5;ministrapp:tin_block->9_ministrapp:chunks#6;ministrapp:torite_block->9_ministrapp:chunks#9;ministrapp:titanium_block->9_ministrapp:chunks#10;ministrapp:desert_quartz->2_ministrapp:desert_quartz_item;ministrapp:desert_quartz_block->4_ministrapp:desert_quartz_item;ministrapp:radiant_quartz_block->4_ministrapp:radiant_quartz;minecraft:wheat->2*ministrapp:dough;ministrapp:copper_ingot->ministrapp:chunks#5;ministrapp:tin_ingot->ministrapp:chunks#6;ministrapp:torite_ingot->ministrapp:chunks#9;ministrapp:titanium_ingot->ministrapp:chunks#10;
}

options {
    # If true, then ore generation will be handled exclusively by oregen .json files 
    # (vanilla ore generation will be disabled) [default: false]
    B:disable_standard_ore_generation=false

```
# If true, then the crack hammer cannot crush ingots into powders if that 
# crackhammer is not hard enough to crush the ingot's ore. [default: true]
B:enforce_hardness=true

# If true, then the crack hammer can crush ingots/ores that a pickaxe of the same 
# material can harvest. If false, then your crack hammer must be made of a harder 
# material than the ore you are crushing. [default: true]
B:strong_hammers=true

# Controls the rarity of metal ingots being found in treasure chests relative to 
# the frequency of other chest loot items. Set to 0 to disable metal ingots from 
# appearing in treasure chests. [range: 0.0 ~ 1000.0, default: 0.5]
S:treasure_chest_loot_factor=0.5
```

}

**Comment by DrPlantabyte** [comment]
Thanks. I'll see what I can do. Does your log file have any error messages from Base Metals? (Search for the phrase "Failed to add recipe")

**Comment by DrPlantabyte** [comment]
I found the problem. There is no item corresponding to ministrapp:desert_quartz_block or ministrapp:radiant_quartz_block and I had a bug that caused incomplete recipes to be added to the hammer registry anyway. I've fixed that bug (in version 1.3.4) so now there is no crash, even if Base Metals cannot find a given item ID.
